<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-T79D26N" failures="1" tests="5" name="SubmitOrder" time="49.204" errors="0" timestamp="2024-08-16T14:26:05 IST">
  <testcase classname="amrutraibagi.Testclasses.SubmitOrderTest" name="SubmitIphoneProduct" time="11.656"/>
  <testcase classname="amrutraibagi.Testclasses.SubmitOrderTest" name="SubmitIphoneProduct" time="10.667"/>
  <testcase classname="amrutraibagi.Testclasses.SubmitOrderTest" name="verifyOrdersPage" time="1.438">
    <skipped/>
  </testcase> <!-- verifyOrdersPage -->
  <testcase classname="amrutraibagi.Testclasses.SubmitOrderTest" name="verifyOrdersPage" time="1.544">
    <skipped/>
  </testcase> <!-- verifyOrdersPage -->
  <testcase classname="amrutraibagi.Testclasses.SubmitOrderTest" name="verifyOrdersPage" time="1.35">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at amrutraibagi.Testclasses.SubmitOrderTest.verifyOrdersPage(SubmitOrderTest.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- verifyOrdersPage -->
</testsuite> <!-- SubmitOrder -->
